tests/commands/test_pipelines_commands.py

```diff
@@ -245,17 +245,14 @@ def test_non_json_api_exception_body(capture_logs, unstub):
         ApiException(
             status=403,
             reason=None,
-            body='403 Forbidden: blah blahbalh',
+            body="403 Forbidden: blah blahbalh",
         )
     )
 
     result = runner.invoke(pipelines_commands.pipelines, ["details", pipeline_name])
 
     assert result.exit_code != 0
     verify(pipelines_commands.pipelines_logic).get_pipeline_info(pipeline_name, None)
-    assert (
-        "403 Forbidden: blah blahbalh"
-        in capture_logs.text
-    )
+    assert "403 Forbidden: blah blahbalh" in capture_logs.text
```

Comment:
```suggestion
    assert "403 Forbidden: blah blah blah" in capture_logs.text
```

I know, I know. But it just bothers me.